packages/react-native/.eslintrc.json

```diff
@@ -37,7 +37,8 @@
               "@nx/webpack",
               "@nx/detox",
               "@nx/cypress",
-              "@nx/playwright"
+              "@nx/playwright",
+              "ajv"
```

Comment:
We shouldn't need to add to ignoredDependencies if its also in the package.json, can you remove?

Reply:
it actually would throw an eslint error when "ajv" is not added: `error  The "ajv" package is not used by "react-native" project  @nx/dependency-checks`